Dart/src/com/jetbrains/lang/dart/ide/errorTreeView/DartProblemsViewPanel.java

```diff
@@ -14,6 +19,13 @@ public DartProblemsViewPanel(Project project) {
   @Override
   protected void fillRightToolbarGroup(DefaultActionGroup group) {
     super.fillRightToolbarGroup(group);
+    group.add(new AnAction(DartBundle.message("dart.reanalyze.action.name"), DartBundle.message("dart.reanalyze.action.description"),
```

Comment:
Thank you, good to know.
I thought I still have to patch enablement somehow, because probably we want to run Reanalyze from the view ignoring what file is selected. Hm... But maybe we can check hosting view in update()? I don't see any related key in CommonDataKeys though.